monitor/extractors/docker_test.go

```diff
@@ -5,8 +5,52 @@ import (
 	"fmt"
 	"os"
 	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"go.aporeto.io/trireme-lib/monitor/constants"
 )
 
+func TestDefaultMetadataExtractor(t *testing.T) {
+	info := &types.ContainerJSON{
+		ContainerJSONBase: &types.ContainerJSONBase{
+			Name:  "name",
+			State: &types.ContainerState{},
+			HostConfig: &container.HostConfig{
+				NetworkMode: constants.DockerHostMode,
+			},
+		},
+		NetworkSettings: &types.NetworkSettings{
+			DefaultNetworkSettings: types.DefaultNetworkSettings{
+				IPAddress: "10.0.0.1",
+			},
+		},
+		Config: &container.Config{
+			Image: "image",
+			Labels: map[string]string{
+				"   ":            "remove me",
+				"empty-label":    "",
+				"standard-label": "one",
+			},
+		},
+	}
+
+	pu, err := DefaultMetadataExtractor(info)
+	if err != nil {
+		t.Error(err)
+	}
+	var foundEmptyTag bool
+	for _, tag := range pu.Tags().Tags {
+		if tag == "@usr:empty-label=<empty>" {
```

Comment:
We have been removing `@usr`tags for a few months, you should not use this prefix anymore

Reply:
@t00f I have to, because this is the indicator in the enforcer to remove it and put it into the associated tags